samples/index.html

```diff
@@ -56,6 +56,7 @@ <h3>Classic editor</h3>
 					v-if="!hide"
 					v-show="show"
 					:read-only="readOnly"
+					:script-loaded="scriptLoaded"
```

Comment:
I think `:namespace-loaded` would make a more accurate name for this property - `script-loaded` isn't very intuitive about what script do we mean. If you agree, please update it everywhere.